Fix parseForESLint is invalid

### Describe the bug

`npm run lint` is not able to validate `NavTrail.svelte`.  

I tried to reinstall the whole project but I am not able to get the linter to work with this file.

### Screenshots

<img width="565" alt="image" src="https://user-images.githubusercontent.com/123702696/219875577-4bf3be78-40a3-4d23-88f2-aa021b6798bb.png">


### Reproduction

Checkout the latest `main` branch and run `npm run lint`

### Logs

```shell
> listd@0.0.1 lint
> eslint --ext .js,.cjs,.ts,.svelte .


Error while parsing ***\NavTrail.svelte
Line 1, column 8: '>' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored
```


### Have you checked if this issue has already been raised?

- [X] I did not find any similar issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

Same error, in my case the console output is

```
Error while parsing /home/---/Github/CodingGarden/listd/src/routes/NavTrail.svelte
Line 1, column 8: '>' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

/home/---/Github/CodingGarden/listd/src/routes/NavTrail.svelte
  44:19  error  Insert `·`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

husky - pre-commit hook exited with code 1 (error)
```

I'm guessing eslint doesn't like the `lang="ts"` in the .svelte file, but even then, if we remove that, it will still complain about some different error.

```
Error while parsing /home/---/Github/CodingGarden/listd/src/routes/NavTrail.svelte
Line 2, column 1: Expression expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

/home/---/Github/CodingGarden/listd/src/routes/NavTrail.svelte
  44:19  error  Insert `·`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

--- a/.eslintrc.cjs
+++ b/.eslintrc.cjs
@@ -37,7 +37,7 @@ module.exports = {
 	},
 	settings: {
 		'import/parsers': {
-			'@typescript-eslint/parser': ['.cjs', '.js', '.ts', '.svelte'],
+			'@typescript-eslint/parser': ['.cjs', '.js', '.ts'],
 		},
 		'import/resolver': {
 			typescript: {

--- a/src/routes/+layout.svelte
+++ b/src/routes/+layout.svelte
@@ -4,9 +4,8 @@
 	import '../app.postcss';
 	import { AppBar, AppShell } from '@skeletonlabs/skeleton';
 	import { setLocale } from '$lib/i18n/i18n-svelte.js';
+	import NavTrail from '$/routes/NavTrail.svelte';
 	import type { LayoutData } from './$types.js';
-
-	import NavTrail from './NavTrail.svelte';
 
 	export let data: LayoutData;
 	setLocale(data.locale);

--- a/src/routes/+page.svelte
+++ b/src/routes/+page.svelte
@@ -1,5 +1,6 @@
 <script lang="ts">
 	import { signIn } from '@auth/sveltekit/client';
+	// TODO: fix load times...
 	import { IconBrandYoutube } from '@tabler/icons-svelte';
 	import { LL } from '$lib/i18n/i18n-svelte';
 	import { page } from '$app/stores';

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,6 +2,7 @@
 	"extends": "./.svelte-kit/tsconfig.json",
 	"compilerOptions": {
 		"paths": {
+			"$/*": ["./src/*"],
 			"$lib": ["./src/lib"],
 			"$lib/*": ["./src/lib/*"],
 			"$app": ["./node_modules/@sveltejs/kit/types"],

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -1,10 +1,16 @@
 import { sveltekit } from '@sveltejs/kit/vite';
 import type { UserConfig } from 'vite';
+import { resolve } from 'path';
 
 const config: UserConfig = {
 	plugins: [sveltekit()],
 	test: {
 		include: ['src/**/*.{test,spec}.{js,ts}'],
+	},
+	resolve: {
+		alias: {
+			$: resolve('./src'),
+		},
 	},
 };
 

